.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -139,6 +139,7 @@ body:
         - 'systemd-tmpfiles'
         - 'systemd-udevd'
         - 'systemd-userdb'
+        - 'systemd-vmspawn'
```

Comment:
Please sort alphabetically, i.e. place vmspawn after veritysetup

Reply:
Whoops, I mixed up the alphabet.